charts/gsp-cluster/templates/02-gsp-system/grafana-concourse-dashboard.yaml

```diff
@@ -1,11 +0,0 @@
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: '{{ .Release.Name }}-grafana-concourse-dashboard'
-  namespace: {{ .Release.Namespace }}
-  labels:
-    grafana_dashboard: "1"
-data:
-  config.json: |-
```

Comment:
Is this extra `-` the problem?

Reply:
The hyphen should just remove a trailing newline I think, but I do wonder if the whole `|-` is the problem

Reply:
TIL.